public/app/plugins/panel/pluginlist/PluginList.tsx

```diff
@@ -0,0 +1,161 @@
+            <h6 className={styles.sectionHeader}>{category.header}</h6>
+            {category.list.map((plugin) => (
+              <a className={styles.item} href={plugin.defaultNavUrl} key={`plugin-${plugin.id}`}>
+                <img src={plugin.info.logos.small} className={styles.image} width="17" height="17" alt="" />
+                <span className={styles.title}>{plugin.name}</span>
+                <span className={styles.version}>v{plugin.info.version}</span>
+                <UserProvider userId={config.bootData.user.id}>
+                  {(api: any, states: any, teams: any, orgs: any, sessions: any, user?: UserDTO) => {
+                    if (!user?.isGrafanaAdmin) {
+                      return <></>;
+                    }
+
+                    return (
+                      <>
+                        {plugin.hasUpdate ? (
+                          <ModalsController>
+                            {({ showModal, hideModal }) => (
+                              <Tooltip content={`New version: ${plugin.latestVersion}`} placement="top">
+                                <span
+                                  className={cx(styles.message, styles.messageUpdate)}
+                                  onClick={(e) => {
+                                    e.preventDefault();
+
+                                    showModal(UpdatePluginModal, {
+                                      pluginID: plugin.id,
+                                      pluginName: plugin.name,
+                                      onDismiss: hideModal,
+                                      isOpen: true,
+                                    });
+                                  }}
+                                >
+                                  Update available!
+                                </span>
+                              </Tooltip>
+                            )}
+                          </ModalsController>
+                        ) : plugin.enabled ? (
+                          <span className={cx(styles.message, styles.messageNoUpdate)}>Up to date</span>
+                        ) : (
+                          <span className={cx(styles.message, styles.messageEnable)}>Enable now</span>
+                        )}
+                      </>
+                    );
+                  }}
+                </UserProvider>
+              </a>
+            ))}
+
+            {category.list.length === 0 && (
+              <a className={styles.item} href="https://grafana.com/plugins">
+                <span className={styles.noneInstalled}>
+                  None installed. <em className={styles.emphasis}>Browse Grafana.com</em>
+                </span>
+              </a>
+            )}
+          </div>
+        ))}
+      </div>
+    </CustomScrollbar>
+  );
+}
+
+const getStyles = stylesFactory((theme: GrafanaTheme) => ({
+  pluginList: css`
+    display: flex;
+    flex-direction: column;
+    gap: ${theme.spacing.d};
```

Comment:
The `gap` flexbox property [isn't supported](https://caniuse.com/flexbox-gap) in the latest Safari, so not sure if we should use it.